packages/cms/src/lokalize/add-text.ts

```diff
@@ -0,0 +1,148 @@
+/**
+ * Add one or multiple texts to the Sanity "Lokalize" dataset.
+ */
+import { set } from 'lodash';
+import prompts from 'prompts';
+import { getClient } from '../client';
+import { appendTextMutation } from './logic';
+import { LokalizeText } from './types';
+
+(async function run() {
+  const client = getClient('development');
+
+  /**
+   * @TODO We could cache the subjects in a temp location on disk and update
+   * them every time this CLI is run. This way we can avoid having to wait for
+   * the query to return.
+   */
+  const allTexts = (await client
+    .fetch(`*[_type == 'lokalizeText']`)
+    .catch((err) => {
+      throw new Error(`Failed to fetch texts: ${err.message}`);
+    })) as LokalizeText[];
+
+  const allSubjects = Object.keys(
+    allTexts.reduce((acc, x) => set(acc, x.subject, true), {})
+  );
+
+  const subjectChoices = Object.keys(
+    allTexts.reduce((acc, x) => set(acc, x.subject, true), {})
+  ).map((x) => ({ title: x, value: x }));
```

Comment:
and here you can use the previous variable
```suggestion
  const subjectChoices = allSubjects.map((x) => ({ title: x, value: x }));
```